prediction_market_agent/agents/microchain_agent/functions.py

```diff
@@ -123,7 +127,24 @@ def __call__(self, market_id: str) -> str:
         question = self.market_type.market_class.get_binary_market(
             id=market_id
         ).question
-        private_key = APIKeys().bet_from_private_key.get_secret_value()
+        result: MechResult = self.mech_request(question, self.mech_tool)
+        return str(result.p_yes)
+
+
+class PredictProbabilityForQuestion(PredictProbabilityForQuestionBase):
```

Comment:
Can we rename this to something like 'PredictProbabilityForQuestionExternal' or similar? Just to differentiate better between local vs remote.